SpiceQL/src/utils.cpp

```diff
@@ -399,11 +399,9 @@ namespace SpiceQL {
     // First try getting the entire state matrix (6x6), which includes CJ and the angular velocity
     checkNaifErrors();
     frmchg_((int *) &refFrame, (int *) &toFrame, &et, (doublereal *) stateCJ);
-    checkNaifErrors();
-
-    if (!failed_c()) {
+    try {
       // Transpose and isolate CJ and av
-      checkNaifErrors();
+      checkNaifErrors(true);
```

Comment:
`reset=true` should be the default behavior. Is there a reason for passing it manually?